source/handshake.c

```diff
@@ -368,6 +374,27 @@ int main(int argc, char * const argv[])
             l_flag = 1;
             test_case = TC_SSL_CTX_POOL;
             break;
+        case 'S': {
+            char *end = NULL;
+            errno = 0;
```

Comment:
The errno assignment should be done after the empty line.

Reply:
Done.